File: NalabsNativeLib/Requirement.cs
﻿namespace Addiva.Nalabs.Interop;


public struct Requirement
{
    public IntPtr Id { get; set; }

    public IntPtr Text { get; set; }

    public double AriScore { get; set; }

    public int Conjunctions { get; set; }

    public int VaguePhrases { get; set; }

    public int Optionality { get; set; }

    public int Subjectivity { get; set; }

    public int References { get; set; }

    public int Weakness { get; set; }

    public int Imperatives { get; set; }

    public int Continuances { get; set; }

    public int Imperatives2 { get; set; }

    public int References2 { get; set; }
}

File: nalabs_shim/se/addiva/nalabs/Requirement.java
package se.addiva.nalabs;

public class Requirement {

    public Requirement(){}

    public Requirement(String id, String text){
        Id = id;
        Text = text;
    }

    public String Id;
    public String Text;
    public double AriScore;
    public int Conjunctions;
    public int VaguePhrases;
    public int Optionality;
    public int Subjectivity;
    public int References;
    public int Weakness;
    public int Imperatives;
    public int Continuances;
    public int Imperatives2;
    public int References2;
}
